src/components/themed/EdgeText.tsx

```diff
@@ -45,7 +45,7 @@ export const Paragraph = (props: {
    * Specific font style definitions for size and color:
    * - default: Default EdgeText style
    * - warning: Orange font color
-   * - finePrint: Smaller, dimmed font
+   * - finePrint: Smaller, orange font
```

Comment:
If it's orange, should it be called "warning" instead? Or even `<FinePrint><WarningText>{message}</WarningText></FinePrint>`?

Sometime the fine print is a warning, and sometimes it isn't, so I personally like the mix & match approach.